Development/src/pages/devices/ChannelMappingMatrix.js

```diff
@@ -48,129 +62,321 @@ const TooltipDivider = withStyles({
     },
 })(Divider);
 
-const getOutputTooltip = (outputId, outputItem, io) => {
+const InteractiveTooltip = ({
+    getTooltip,
+    display,
+    tooltipOpen,
```

Comment:
This means not "is open" but "stay open" if I understand correctly?
Just thinking about this after playing with the UI now this is implemented, maybe we actually just want to know whether an interactive element in the tooltip currently has the focus, and keep it open while that is the case. I.e. maybe clicking off a tooltip once you start editing should cancel the edit?
Need to think more!

Reply:
I hope this behaviour will be achievable by [detecting focus entering and leaving the component tree] (https://reactjs.org/docs/events.html#detecting-focus-entering-and-leaving).

Reply:
lets talk, I don't understand exactly what do you mean :)

Reply:
I added commit that handling with this issue. 
when the user click away the tooltip (on edit mode) , the tooltip disappeared.

Reply:
Renamed as `tooltipModal` to indicate that it means an interactive tooltip is currently modal.

Reply:
FWIW, `ClickAwayListener` now moved inside the tooltip content which means we don't get tens or hundreds of them firing!